oss-internship-2020/gdal/raster_to_gtiff/gtiff_converter.cc

```diff
@@ -0,0 +1,126 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "gtiff_converter.h"  // NOLINT(build/include)
+
+namespace gdal::sandbox {
+
+namespace {
+
+  inline constexpr absl::string_view kDriverName = "GTiff";
+
+}  // namespace
+
+RasterToGTiffProcessor::RasterToGTiffProcessor(std::string out_file_full_path, 
+                                                std::string out_file_folder,
```

Comment:
should be possible to get it from `out_file_path`
`file_util::fileops::StripBasename`